packages/manager/apps/pci-databases-analytics/src/pages/services/create/_components/ApiTerraformDialog.component.tsx

```diff
@@ -0,0 +1,315 @@
+    }
+  };
+
+  // Fill API configuration
+  const apiBody: ServiceCreationWithEngine = {
+    description: dialogData.name,
+    engine: dialogData.availability.engine as database.EngineEnum,
+    nodesPattern: {
+      flavor: dialogData.availability.flavor,
+      number: dialogData.nodes,
+      region: dialogData.availability.region,
+    },
+    plan: dialogData.availability.plan,
+    version: dialogData.availability.version,
+    ipRestrictions: dialogData.ipRestrictions,
+  };
+  if (dialogData.network.type === database.NetworkTypeEnum.private) {
+    const networkOpenstackId = dialogData.network.network.regions.find((r) =>
+      r.region.includes(dialogData.availability.region),
+    ).openstackId;
+    apiBody.networkId = networkOpenstackId;
+    apiBody.subnetId = dialogData.network.subnet.id;
+  }
+  if (dialogData.flavor.storage) {
+    apiBody.disk = {
+      size:
+        dialogData.flavor.storage.minimum.value + dialogData.additionalStorage,
+    } as database.service.Disk;
+  }
+  // Destructure body to extract engine and keep the rest
+  const { engine, ...formattedApiBody } = apiBody;
+  const apiEndpoint = `/cloud/project/${projectId}/database/${engine}`;
+  const apiBodyContent = formattedApiBody;
+
+  return (
+    <>
+      <Dialog open={true} onOpenChange={onRequestClose}>
+        <DialogContent className="fixed left-1/2 top-1/2 max-h-[80vh] w-[90vw] max-w-xl -translate-x-1/2 -translate-y-1/2 p-4">
+          <DialogHeader>
+            <DialogTitle>
+              {t('pciDatabasesAddCommandDialogEquivalentOrder')}
+            </DialogTitle>
+          </DialogHeader>
+          <Tabs
+            defaultValue={currentTab}
+            value={currentTab}
+            onValueChange={handleTabChange}
+          >
+            <TabsList className="grid grid-cols-2">
+              <TabsTrigger value="api" data-testid="api-tab">
+                {t('pciDatabasesAddCommandDialogApiTabTitle')}
+              </TabsTrigger>
+              <TabsTrigger value="terraform" data-testid="terraform-tab">
+                {t('pciDatabasesAddCommandDialogTerraformTabTitle')}
+              </TabsTrigger>
+            </TabsList>
+            <TabsContent
+              value="api"
+              className="max-h-[60vh] overflow-auto space-y-4"
+            >
+              <p>{t('pciDatabasesAddCommandDialogApiDescription')}</p>
+              <Trans
+                t={t}
+                i18nKey={'pciDatabasesAddCommandDialogDocumentationCta'}
+                components={{
+                  anchor: (
+                    <A
+                      target="_blank"
+                      href={getDocumentationUrl(LINKS.DB_ORDER_EQ_API, locale)}
+                    ></A>
+                  ),
+                }}
+              ></Trans>
+              <div>{t('pciDatabasesAddCommandDialogApiParameters')}</div>
+              <div className="inline-flex items-center gap-1 font-mono text-sm">
```

Comment:
Maybe it would look better having the items top-aligned
```suggestion
              <div className="inline-flex gap-1 font-mono text-sm items-start">
```